platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,114 @@
+
+On Seqera Cloud, the free tier permits only one running data studio session at a time. To run simultaneous data studios, [contact Seqera][contact] for a Seqera Cloud Pro license.
+
+:::note
+Data Studios is currently in **public preview** and is available from Seqera Platform v24.1. If you experience any problems during the deployment process please contact your account executive. Data Studios in Enterprise is not enabled by default. You can enable Data Studios in the [environment variables configuration](../enterprise/data-studios.mdx).
+:::
+
+Select the **Data Studios** tab in Platform to view all data studios. The list includes the name, cloud provider, analysis template, region, author, creation date, and the status of each session. In this view, you can add a new data studio session, as well as start, stop, and connect to an existing data studio session. You can dynamically filter the list of data studio sessions using the search bar and searching by name (default), author username, or compute environment name. Selecting the session name opens a detailed view of the data studio, displaying configuration information.
+
+## Requirements
+
+Before you get started, ensure you have the following:
+
+- Valid credentials to access your cloud storage data resources.
+- At least the **Maintain** role set of permissions.
+- A compute environment with sufficient resources. This is highly dependent on the volume of data you wish to process, but we recommended at least 2 CPUs allocated with 8192 MB of memory. See [AWS Batch](../compute-envs/aws-batch.mdx) for more information about compute environment configuration.
+- [Data Explorer](../data/data-explorer.mdx) is enabled.
+
+:::note
+Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
+:::
+
+## Container image templates
+
+Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the data studio session container is built. Users have permission to install additional packages as needed.
+
+In addition, you can create your own container image templates. For more information, see [Custom environments][custom-envs].
+
+**JupyterLab**
+
+The default user is the `root` account. The following [conda-forge](https://conda-forge.org/) packages are available by default:
+
+- `altair`
+- `ipywidgets`
+- `itables`
+- `jedi-language-server`
+- `jupyter-collaboration`
+- `jupyter-dash`
+- `jupyterlab-git`
+- `jupyterlab`
+- `jupytext`
+- `nb_black`
+- `pandas[all]`
+- `pip`
+- `plotly`
+- `python`
+- `qgrid`
+- `r-ggplot2`
+- `scikit-learn`
+- `seaborn[stats]`
+- `statsmodels`
```

Comment:
@t0randr We should probably pin these to the most recent stable versions and note here so it's clear what we provide out-of-the-box.

Reply:
These are indeed pinned: https://github.com/seqeralabs/connect/blob/967a5316d3d90708281123ef82c08d1bec5a0741/docker/jupyter_4.2.5/env.yaml

I think we should also add the jupyterlab version: `4.2.5` (not yet released in prod)